app-emulation/punes/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+<email>azamat.hackimov@gmail.com</email>
```

Comment:
should also states gentoo proxy-maint team here, as you are not a developer.

Reply:
@Zlogene done.